src/lib/CookieJar.js

```diff
@@ -50,17 +52,19 @@ class CookieJar { // eslint-disable-line no-unused-vars
     static _decodeCookie(str) {
         if (!str) return [];
 
-        if (str.length % 15 !== 0) throw new Error('Malformed cookie');
+        if (str.length % 14 !== 0) throw new Error('Malformed cookie');
 
-        const keys = str.match(/.{15}/g);
+        const keys = str.match(/.{14}/g);
```

Comment:
I think if the cookie is empty, because all keys where logged-out of, then it would maybe return NULL? I think the check for NULL is actually wanted now, and the comment about TS can be removed.